src/compileModel.ts

```diff
@@ -164,12 +182,16 @@ const compileModel = <TSchema extends GenericObject = GenericObject>(
 			id: string,
 			options: ModelFindByIdOptions = {},
 		): Promise<Model | null> {
-			const { projection = [] } = options;
-			const data = await Model.connection.executeDbFeature('findById', {
-				filename: Model.file,
-				id,
-				projection: Model.schema?.transformPathsToDbPositions(projection) ?? [],
-			});
+			const { projection = [], userDefined } = options;
+			const data = await Model.connection.executeDbFeature(
+				'findById',
+				{
+					filename: Model.file,
+					id,
+					projection: Model.schema?.transformPathsToDbPositions(projection) ?? [],
+				},
+				userDefined && { userDefined },
```

Comment:
Is there any use for these values when calling `find`? No non-mvom code can react to it. I suppose we could implement some kind of server side logging in the future where we could log these user defined values alongside whatever else we decide we want to log?

I wouldn't be opposed to only passing these to the save and delete routines.

Reply:
I can't think of a use case for anything other than save and delete.  I don't know of any way to access them upstream except in triggers, and triggers are only fired on update and delete operations.  I guess if we ever added additionally functionality for these variables other than for adding to common it might come in handy.  I can remove it if you think its superfluous.

Reply:
I suppose it's harmless.